samples/kubernetes/deploy-aks.sh

```diff
@@ -0,0 +1,214 @@
+#!/bin/bash
+
+set -eE -o functrace
+
+failure() {
+  local lineno=$1
+  local msg=$2
+  echo -e "\033[31mScript failed at $lineno: ${msg}\e[0m"
+}
+
+trap 'failure ${LINENO} "$BASH_COMMAND"' ERR
+
+usage() {
+    echo "deploy-aks.sh  --resource-group-name <group name>"
+    echo "              [--location <location>]"
+    echo "              [--max-nodes <default 5> ]"
+    echo "              [--vm-size <default Standard_DS2_v2>]"
+    echo "              [--help]"
+}
+
+# Checking if we have the tools we need to run this script
+command -v jq >/dev/null 2>&1 || { echo >&2 "'jq' is required but not installed. Aborting."; exit 1; }
+command -v helm >/dev/null 2>&1 || { echo >&2 "'helm' (version 3) is required but not installed. Aborting."; exit 1; }
+command -v az >/dev/null 2>&1 || { echo >&2 "'az' (Azure CLI) is required but not installed. Aborting."; exit 1; }
+command -v kubectl >/dev/null 2>&1 || { echo >&2 "'kubectl' is required but not installed. Aborting."; exit 1; }
+
+# Settings
+location="westus2"
+maxNodes=5
+vmSize="Standard_DS2_v2"
+environmentName=""
+
+# Parse command line arguments
+while [ "$1" != "" ]; do
+    case $1 in
+        -e | --environment-name )                   shift
```

Comment:
I think we should have --name and --resource-group instead of this.

Reply:
Done